pkg/runner/runner.go

```diff
@@ -70,6 +70,7 @@ func (runner *runnerImpl) NewPlanExecutor(plan *model.Plan) common.Executor {
 
 			for i, matrix := range matrixes {
 				rc := runner.newRunContext(run, matrix)
+				rc.JobName = rc.Name
```

Comment:
Why and what does it fix?

Reply:
https://github.com/jsoref/gha-debug/blob/main/.github/workflows/debug.yml should show `build`

Reply:
I'd prefer this change (and other lines part of fix) to be separate commit with clear description of fixed issue.
It helps later while looking through history.